nixosModules/hermes/base.nix

```diff
@@ -0,0 +1,18 @@
+{
+  lib,
+  nix-std,
+  cfg,
+}:
+with lib; {
+  options = {
+    config = import ./config-options.nix {inherit lib;};
+    toml = mkOption {type = types.unique {message = "only one toml output";} types.str;};
+  };
+  config = {
+    toml = let
+      # remove `null` from toml render
+      sanitizedCfg = filterAttrsRecursive (_: v: v != null) cfg.config;
```

Comment:
Why do we need `toml` twice?

Reply:
Oh right, I missed that. Sometimes it gets confusing between module options and actual module configuration. Thanks for clarifying.